webapp/javascript/components/Settings/Security/ChangePasswordForm.tsx

```diff
@@ -66,6 +66,7 @@ function ChangePasswordForm(props: ShamefulAny) {
             name="oldPassword"
             required
             onChange={handleChange}
+            value={form.oldPassword}
```

Comment:
how was this working without these before? What different functionality does adding them now do?

Reply:
technically it's possible, text input can render it's value without passing value to it, I prefer passing value to all inputs

`<input type="text" /> == <input type="text" value={value}>`